internal/cmd/secrets-manager/user/update/update.go

```diff
@@ -0,0 +1,135 @@
+package update
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/stackitcloud/stackit-cli/internal/pkg/args"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/confirm"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/errors"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/examples"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/flags"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/globalflags"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/services/secrets-manager/client"
+	secretsManagerUtils "github.com/stackitcloud/stackit-cli/internal/pkg/services/secrets-manager/utils"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/utils"
+
+	"github.com/spf13/cobra"
+	"github.com/stackitcloud/stackit-sdk-go/services/secretsmanager"
+)
+
+const (
+	userIdArg = "USER_ID"
+
+	instanceIdFlag   = "instance-id"
+	enableWriteFlag  = "enable-write"
+	disableWriteFlag = "disable-write"
+)
+
+type inputModel struct {
+	*globalflags.GlobalFlagModel
+
+	InstanceId   string
+	UserId       string
+	EnableWrite  *bool
+	DisableWrite *bool
+}
+
+func NewCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   fmt.Sprintf("update %s", userIdArg),
+		Short: "Updates the write privileges Secrets Manager user",
+		Long:  "Updates the write privileges Secrets Manager user.",
+		Example: examples.Build(
+			examples.NewExample(
+				`Enable write access of a Secrets Manager user with ID "xxx" of instance with ID "yyy"`,
+				"$ stackit secrets-manager user update xxx --instance-id yyy --enable-write"),
+			examples.NewExample(
+				`Disable write access of a Secrets Manager user with ID "xxx" of instance with ID "yyy"`,
+				"$ stackit secrets-manager user update xxx --instance-id yyy --disable-write"),
+		),
+		Args: args.SingleArg(userIdArg, utils.ValidateUUID),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			ctx := context.Background()
+			model, err := parseInput(cmd, args)
+			if err != nil {
+				return err
+			}
+
+			// Configure API client
+			apiClient, err := client.ConfigureClient(cmd)
+			if err != nil {
+				return err
+			}
+
+			instanceLabel, err := secretsManagerUtils.GetInstanceName(ctx, apiClient, model.ProjectId, model.InstanceId)
+			if err != nil {
+				instanceLabel = model.InstanceId
+			}
+
+			userLabel, userDescription, err := secretsManagerUtils.GetUserDetails(ctx, apiClient, model.ProjectId, model.InstanceId, model.UserId)
+			if err != nil {
+				userLabel = model.UserId
+			}
```

Comment:
The idea of this `...Label` where we recover from the error is that we have a backup in case the API call to get the details fails (instead of making the command return an error for this nice-to-have).

Which means that the `userLabel` should be either `<user name> (<description>)` (nice-to-have) or `<user ID>` (in case the API call fails)